src/overlays/actors/ovl_En_Test6/z_en_test6.h

```diff
@@ -7,10 +7,46 @@ struct EnTest6;
 
 typedef void (*EnTest6ActionFunc)(struct EnTest6*, GlobalContext*);
 
+#define ENTEST6_GET(thisx) ((thisx)->params)
+
+enum {
+    /* 24 */ ENTEST6_24 = 24,
+    /* 25 */ ENTEST6_25,
+    /* 26 */ ENTEST6_26,
+};
```

Comment:
This is the format I've always followed. params are ACTOR_GET_BITS. In this case there are no specific bits so I dorp the _BITS. For their values I always use ACTOR_BITS_VALUE. No bits so they're dropped and I'm left with ACTOR_VALUE. I've done all my actors this way since it's the way Kenix said to name them. I don't have proper names because I'm not documentating, so I'm not really sure what you want me to change here.